### PR TITLE
V4 docs

### PR DESCRIPTION
**Pull request by WoodNeck**
## Issue
#409

### CODE REVIEW

File: src/control/AxesController.ts [hunk 1]
```diff
@@ -64,8 +117,15 @@ class AxesController {
...
       });
     }
+
+    return this;
   }
 
+  /**
+   * Destroy AxesController and return to initial state
+   * @ko AxesController를 초기 상태로 되돌립니다
+   * @return {void}
```
Comment (daybrush) [inline src/control/AxesController.ts:127]: @WoodNeck 

it's `return {this}`

**Review by daybrush — APPROVED** [review]
LGTM

**Review by mixed — APPROVED** [review]
👏👏👏

File: src/control/AxesController.ts (hunk 1 above)
Comment (WoodNeck) [reply src/control/AxesController.ts:127]: Thanks~ I'll fix that

**Pull request merged**